pkg/os/exec.go

```diff
@@ -48,6 +48,9 @@ func RunPrivileged(reason string, cmdAndArgs ...string) (string, string, error)
 	if err != nil {
 		return "", "", errors.New("sudo binary not found")
 	}
+	if _, _, err := run(sudo, []string{"--validate"}, map[string]string{}); err != nil {
+		return "", "", errors.New("user may not use sudo")
+	}
```

Comment:
This feels like something which should be part of a preflight check rather than something we run every time `RunPrivileged` is called.

Reply:
Yes but if everything is ok in crc setup, sudo is not run at all. If I change it to a preflight, it will always force the user to type his password.

Reply:
Since you want to make this error 'special' («It will help us group this kind of errors under the same name in the telemetry. ») , might be worth having a constant for it, or even its own error type. 
`"user is not allowed to use the 'sudo' command"` or somethnig like this rather than `may not use`?

Reply:
yes, all done!